public/app/features/explore/Explore.tsx

```diff
@@ -410,7 +411,7 @@ export class Explore extends React.PureComponent<ExploreProps, ExploreState> {
                             // If there is not data (like 404) we show a separate error so no need to show anything here
                             queryResponse.series[0] && (
                               <TraceView
-                                trace={queryResponse.series[0].fields[0].values.get(0) as any}
+                                trace={queryResponse.series[0].fields[0].values.get(0) as TraceViewData | undefined}
```

Comment:
just a small nit: why do we need the typings here? Is this something that could be typed further down the chain?

Reply:
What's "down" for you? The producer or the consumer?

Reply:
If we could get a typed queryResponse, that would be optimal, I think.

Reply:
The problem is that we do not exactly know what is in that dataFrame until we are in this component and we are deciding what to show so it would be more work to type that dataFrame.field properly and do all the checks in between. We should probably do that at some point but a bit too much work for this PR.